JavaScript/ArrayMethod/js-array-methods_for-each/js/index.js

```diff
@@ -19,4 +19,19 @@ const colors = [
   "#A5A5A5",
 ];
 
-colors.forEach();
+colors.forEach((color)=> {
+  const hex = document.createElement("div");
+  hex.classList.add('color-box');
+  hex.style.background = color;
+  document.body.append(hex);
+});
+
+
+function renderColorBox(color){
+  const hex = document.createElement("div");
+  hex.classList.add('color-box');
+  hex.style.background = color;
+  document.body.append(hex);
```

Comment:
A template string could be a more elegant solution. Especially if you would have to add more nodes, classes, attributes, etc.